systest/postmortenHandler/postmortenhandler.go

```diff
@@ -0,0 +1,161 @@
+package postmortenHandler
+
+import (
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/systest/parameters"
+	"go.uber.org/zap"
+)
+
+/*
+how the post-morten debug handler works:
+1. As soon the namespace is registered. The user has PostMortenConfirmationWindow
+Minutes to access the post-morten debug handler and perform POST request to notify the handler that the namespace is being debugged.
+That will extend the expiration time to PostMortenConfirmationWindow + PostMortenDebugDuration.
+2. if not the namespace will be cleaned up.
+3. The user performing the POST request to extend the expiration time to Now + the time passed on the request.
+But that time cannot exceed PostMortenDebugMaxDuration.
+4. The user can perform a GET request to check the status of the namespace.
+5. The user can perform a DELETE request to clean up the namespace.
+*/
+type PostMortenDebugHandler struct {
+	Namespace         string
+	isBeingDebugged   bool
+	expriationTime    int64
+	createTime        int64
+	maximumDuration   int64
+	cleaningFunctions chan func()
+	param             *parameters.Parameters
+}
+
+var (
+	Starter          sync.Once
+	server           *http.Server
+	Log              *zap.SugaredLogger
+	namespaces       = map[string]*PostMortenDebugHandler{}
+	lazyRegistration = []map[string]func(){}
+
+	// input parameters
+	PosTMortenDebug = parameters.Bool(
+		"post-morten-debug", "if true post-morten debug is allowed",
+	)
```

Comment:
NIT: also for the other parameters

```suggestion
	PotMortemDebug = parameters.Bool(
		"post-mortem-debug", "if true post-mortem debug is allowed",
	)
```